app/components/job/details/job-configuration.html

```diff
@@ -85,6 +85,10 @@
                     <bl-tp-cell [value]="file.filePath"></bl-tp-cell>
                 </bl-table-property-row>
             </bl-table-property>
+            <bl-text-property *ngIf="managerTask.containerSettings.count > 0" label="Container image name" [value]="managerTask.containerSettings.imageName"></bl-text-property>
+            <bl-text-property *ngIf="managerTask.containerSettings.count > 0" label="Container run option" [value]="managerTask.containerSettings.containerRunOptions"></bl-text-property>
+            <bl-text-property *ngIf="managerTask.containerSettings.count > 0" label="Registry server" [value]="managerTask.containerSettings.registryServer"></bl-text-property>
+            <bl-text-property *ngIf="managerTask.containerSettings.count > 0" label="Registry username" [value]="managerTask.containerSettings.registryUsername"></bl-text-property>
         </bl-property-group>
```

Comment:
wouldn't these be better as: *ngIf="managerTask.containerSettings?.imageName" rather than count. 
count makes me think it's a collection of something.

Reply:
Actually the point is not to show the input if there is no value right?
In that case though I would wrap all the options that needs the same condition under a `ng-container` tag and put the ngif there. `ng-container` is for the angular template only it doesn't get added to the html after.